### PR TITLE
Fix bugs in AddSequenceDiagram and AddActivityDiagram

### PR DESCRIPTION
**Pull request by johnrhimawan**
Currently, the sequence diagram and activity diagram for `add` was for a past version of InTrack, let's change it to reflect InTrack's current version.

### CODE REVIEW

**Review by kangqiao322 — COMMENTED** [review]
LGTM!

File: docs/diagrams/AddSequenceDiagram.puml [hunk 1]
```diff
@@ -14,10 +14,10 @@ box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("add n/Google p/SWE hp/65025300002 \ne/support@google.com a/16 Park Way t/Urgent")
+[-> LogicManager : execute("add c/Google p/SWE \ne/support@google.com w/https://careers.google.com/ \n s/5000 Park Way t/Urgent")
```
Comment (jetlfj) [inline docs/diagrams/AddSequenceDiagram.puml:17]: The `Salary` should be just `5000`, not `5000 Park Way`

File: docs/diagrams/AddSequenceDiagram.puml [hunk 2]
```diff
@@ -14,10 +14,10 @@ box Model MODEL_COLOR_T1
...
 activate LogicManager
 
-LogicManager -> InTrackParser : parseCommand("add n/Google p/SWE hp/65025300002 \ne/support@google.com a/16 Park Way t/Urgent")
+LogicManager -> InTrackParser : parseCommand("add c/Google p/SWE \ne/support@google.com w/https://careers.google.com/ \n s/5000 Park Way t/Urgent")
```
Comment (jetlfj) [inline docs/diagrams/AddSequenceDiagram.puml:20]: Same as above

File: docs/diagrams/AddSequenceDiagram.puml [hunk 3]
```diff
@@ -27,7 +27,7 @@ activate AddCommandParser
 AddCommandParser --> InTrackParser
 deactivate AddCommandParser
 
-InTrackParser -> AddCommandParser : parse("add n/Google p/SWE hp/65025300002 \ne/support@google.com a/16 Park Way t/Urgent")
+InTrackParser -> AddCommandParser : parse("add c/Google p/SWE \ne/support@google.com w/https://careers.google.com/ \n s/5000 Park Way t/Urgent")
```
Comment (jetlfj) [inline docs/diagrams/AddSequenceDiagram.puml:30]: Same as above

File: docs/diagrams/AddSequenceDiagram.puml (hunk 1 above)
Comment (johnrhimawan) [reply docs/diagrams/AddSequenceDiagram.puml]: Thank you for the suggestion.

File: docs/diagrams/AddSequenceDiagram.puml (hunk 2 above)
Comment (johnrhimawan) [reply docs/diagrams/AddSequenceDiagram.puml]: Thank you for the suggestion, I have changed it.

File: docs/diagrams/AddSequenceDiagram.puml (hunk 3 above)
Comment (johnrhimawan) [reply docs/diagrams/AddSequenceDiagram.puml]: Thank you for the advice, it has been implemented.

**Pull request merged**